hudi-common/src/main/java/org/apache/hudi/common/model/OverwriteWithLatestAvroPayload.java

```diff
@@ -90,4 +94,35 @@ public Boolean overwriteField(Object value, Object defaultValue) {
   public Comparable<?> getOrderingValue() {
     return this.orderingVal;
   }
+
+  /**
+   * Returns whether the given record is newer than the record of this payload.
+   *
+   * @param orderingVal
+   * @param record The record
+   * @param prop   The payload properties
+   *
+   * @return true if the given record is newer
+   */
+  protected static boolean isRecordNewer(Comparable orderingVal, IndexedRecord record, Properties prop) {
```

Comment:
is it needed to set `static` modifier?

Reply:
don't need it, I have already removed it.